src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
```

Comment:
Prefer using `const` instead of `let` for the `result` variable since its value is not being reassigned but only modified. This helps to prevent unintentional reassignment and makes the code more robust. See checklist item 3.